### PR TITLE
inline brand related assets

### PR DESCRIPTION
**Pull request by steven-klein**
# What this does

- inlines a fallback png type favicon for Safari
- inlines a primary SVG favicon for the cool kids
- inlines the primary XYPN logo in templates
- remove manifest reference (don't have a manifest to serve - something we can do on a rainy day)
- remove theme-color meta (I'm not opposed to having this, it was just only in the maintenance and error templates and seemed odd.  It's another small touch we can revisit on a rainy day, like apple-touch-icon)

### CODE REVIEW

**Review by profsmallpine — APPROVED** [review]
inline ftw

File: ranger/tmpl/error.tmpl [hunk 1]
```diff
@@ -36,7 +35,9 @@
     padding: .25rem .5rem;
   }
   #logo {
-    margin: 0 0 2rem 100px; 
+    margin: 2rem 120px; 
+    width: 150px;
+    height: auto;
   }
   #sash {
     background: #1F6DF4;
```
Comment (DavidLarsKetch) [inline ranger/tmpl/error.tmpl:43]: Since you're already in here, want to swap the old blue for the new `#2850BE`?

**Review by DavidLarsKetch — APPROVED** [review]
LGTM!

File: ranger/tmpl/error.tmpl (hunk 1 above)
Comment (steven-klein) [reply ranger/tmpl/error.tmpl:43]: good catch!

**Pull request merged**